src/features/game/events/landExpansion/harvestBeehive.ts

```diff
@@ -86,6 +87,10 @@ export const getHoneyMultiplier = (game: GameState) => {
     multiplier -= 0.5;
   }
 
+  if (isCollectibleBuilt({ name: "King of Bears", game })) {
```

Comment:
My read of the boost description is that you only get the boost when full.  The implementation of multiplier applies even for non-full.

Is there an additional if check needed or is the boost better than (my read of) the description?

Reply:
I think the description means that on full bee hive harvest we get additional +0.25 yield in Honey. so on half-full it will be 0.125+. I don't think we need additional checks as this description also applies with Bee Suit and Honeycomb Shield.